features.yaml

```
@@ -240,7 +240,7 @@ features:
     link: "/docs/tasks/security/authorization/authz-custom"
     level:
       checklist: ""
```

Update to point to checklist
```suggestion
      checklist: features/external_authorization.md
```